docs/addons/karpenter.md

```diff
@@ -38,37 +40,32 @@ const karpenterAddOn = new blueprints.addons.KarpenterAddOn({
       annotations: {
           "eks-blueprints/owner": "young"
```

Comment:
I know it is not authored by you, but let' replace the owner with something generic, e.g. "platform-team"